randovania/games/prime3/logic_database/header.txt

```diff
@@ -35,6 +35,13 @@ Templates
 * Hazard Shield:
       Suit Type
 
+* Fight Ghor-G:
+      All of the following:
+          Morph Ball
+          Any of the following:
+              Morph Ball Bomb or Boost Ball
+              Hyper Ball and Knowledge (Beginner) and Enter Hypermode
```

Comment:
I wonder if this should be combat instead?

Reply:
Knowledge is there to know Ghor's weak point can be damaged with Hyper Ball. Not sure why Combat would be better suited, but if you have an idea for why feel free to share.

Reply:
Nah this is fine